extensions/infinispan-embedded/runtime/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <artifactId>quarkus-infinispan-embedded</artifactId>
     <name>Quarkus - Infinispan - Embedded - Runtime</name>
-    <description>Run an embedded Infinispan data grid server for distributed caching</description>
```

Comment:
I'm a bit late to the party but isn't this going to be a problem with code.quarkus.io once we make this extension public? @maxandersen am I right?

Reply:
There's no plan to make this public. It's more likely that we'll move it to a separate repository.

Reply:
whats the problem you for see @gsmet  ? I feel like i'm missing some context ;)

Reply:
To be honest I am not sure why this line was removed. I am more than fine adding it back in if it fixes something :)

Reply:
ooh - now i get it. yes you'll want to make sure you have a description either here or in quarkus-extension.yaml as otherwise it will show up blank on code.quarkus